stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/common/feature/messages/list/MessageListController.kt

```diff
@@ -459,6 +463,11 @@ public class MessageListController(
                             focusedMessage = focusedMessage,
                             unreadLabel = unreadLabel,
                             members = members,
+                            endOfOlderMessages = endOfOlderMessages,
+                            channel = channel.copy(
```

Comment:
Is there any reason why we cannot do `channelState.toChannel()` right here? 🤔 

`channelState.toChannel` internally takes all info from StateFlows, so I was thinking if we gonna keep stale data.

Reply:
`toChannel()` method performs multiple instance creations, copies of the instance, and lists conversions. I wanted to avoid performance issues by calculating data only needed when the user goes to the beginning of the conversation.